pkg/resolver/searchSchema_test.go

```diff
@@ -12,24 +12,39 @@ import (
 )
 
 func Test_SearchSchema_Query(t *testing.T) {
+	searchInput := &model.SearchInput{}
 	// Create a SearchSchemaResolver instance with a mock connection pool.
-	resolver, _ := newMockSearchSchema(t)
+	resolver, _ := newMockSearchSchema(t, searchInput, rbac.UserData{CsResources: []rbac.Resource{}}, nil)
 
 	resolver.userData = rbac.UserData{CsResources: []rbac.Resource{}}
 	sql := `SELECT DISTINCT "prop" FROM (SELECT jsonb_object_keys(jsonb_strip_nulls("data")) AS "prop" FROM "search"."resources" WHERE ("cluster" = ANY ('{}')) LIMIT 100000) AS "schema"`
 	// Execute function
 	resolver.buildSearchSchemaQuery(context.TODO())
 
 	// Verify response
-	if resolver.query != sql {
-		t.Errorf("Expected sql query: %s but got %s", sql, resolver.query)
-	}
+	assert.Equalf(t, sql, resolver.query, "Expected sql query: %s but got %s", sql, resolver.query)
+}
+
+func Test_SearchSchema_Query_WithFilters(t *testing.T) {
+	value1 := "openshift"
+	searchInput := &model.SearchInput{Filters: []*model.SearchFilter{{Property: "namespace", Values: []*string{&value1}}}}
+	propTypesMock := map[string]string{"namespace": "string"}
+	// Create a SearchSchemaResolver instance with a mock connection pool.
+	resolver, _ := newMockSearchSchema(t, searchInput, rbac.UserData{CsResources: []rbac.Resource{}}, propTypesMock)
+
+	resolver.userData = rbac.UserData{CsResources: []rbac.Resource{}}
+	sql := `SELECT DISTINCT "prop" FROM (SELECT jsonb_object_keys(jsonb_strip_nulls("data")) AS "prop" FROM "search"."resources" WHERE ("data"->'namespace'?('openshift') AND ("cluster" = ANY ('{}'))) LIMIT 100000) AS "schema"`
+	// Execute function
+	resolver.buildSearchSchemaQuery(context.Background())
+
+	// Verify response
+	assert.Equalf(t, sql, resolver.query, "Expected sql query: %s but got %s", sql, resolver.query)
```

Comment:
The assert.Equal() function takes care of printing actual and expected, so you can simplify this.
```suggestion
	assert.Equal(t, sql, resolver.query)
```